webf/lib/src/css/matrix.dart

```diff
@@ -609,21 +609,37 @@ class CSSMatrix {
 
   static Matrix4? _computeMatrix(CSSFunctionalNotation method, RenderStyle renderStyle) {
     final methodName = method.name.toLowerCase();
+
+    // resolve var arg
+    var methodArgs = method.args;
+    for (int i = 0; i < methodArgs.length; i++) {
+      String arg = methodArgs[i];
+      final calcValue = CSSCalcValue.tryParse(renderStyle, '', arg);
+      if (calcValue != null) {
+        methodArgs[i] = calcValue.computedValue('').toString();
+      } else {
+        final varValue = CSSVariable.tryParse(renderStyle, arg);
+        if (varValue != null) {
+          methodArgs[i] = varValue.computedValue('');
```

Comment:
type 'Null' is not a subtype of type 'String'

Reply:
👌